docs/architecture/porting-existing-aspnet-apps/incremental-migration-strategies.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Strategies for migrating incrementally
+description: What strategies can teams adopt that will allow them to migrate large apps from ASP.NET MVC to .NET Core in an incremental fashion?
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Strategies for migrating incrementally
+
+The biggest challenge with migrating any large app is determining how to break the process into smaller tasks. There are several strategies that can be applied for this purpose, including breaking the app into horizontal layers such as UI, data access, business logic, or breaking up the app into separate, smaller apps. Another strategy is to upgrade some or all of the app to different framework versions on the way to a recent .NET Core release.
+
+Consider the challenge of migrating a large ASP.NET 4.5 app. One approach is to migrate directly from .NET Framework 4.5 to .NET Core 3.1. However, this approach needs to account for every breaking change between the two frameworks and versions, which are substantial.
+
+One recent addition to the .NET ecosystem that helps with interoperability between different .NET frameworks is [.NET Standard](https://dotnet.microsoft.com/platform/dotnet-standard). .NET Standard allows libraries to build against the agreed upon set of common APIs, ensuring they can be used in any .NET app. .NET Standard 2.0 is notable because it covers most base class library functionality used by most .NET Framework and .NET Core apps. Unfortunately, the earliest version of .NET with support for .NET Standard 2.0 is .NET Framework 4.6.1.
+
+One approach to incrementally upgrade a .NET Framework 4.5 system is to first update its class libraries to .NET Framework 4.6.1. Then, modify these libraries to be .NET Standard class libraries. Use multi-targeting and conditional compilation, if necessary. This step can be helpful in scenarios where app dependencies require .NET Framework and cannot easily be ported directly to use .NET Standard and .NET Core. Since .NET Framework libraries can be consumed by ASP.NET Core 2.1 apps, the next step is to migrate some or all of the web functionality of the app to ASP.NET Core 2.1 (as described in the [previous chapter](choose-net-core-version.md)).
```

Comment:
Given that .NET Framework 4.8 is highly compatible, and there are some known issues with deployment of .NET Standard 2.0 binaries on .NET Framework versions before 4.7.2, I would tend to recommend people update all the way to 4.8 first.